docs/source/author/topics.rst

```diff
@@ -0,0 +1,685 @@
+Topics application
+##############################################################################
+
+The topics application is the main focus of the CS Unplugged website, as it
+contains the majority of educational material for the project.
+
+.. contents:: Contents
+  :local:
+
+Topics overview
+==============================================================================
+
+A general overview of the topics application can be described in the following
```

Comment:
have we defined "application" by this point/does it need to be on this page/does this need to be reworded?